projects/extension/src/background/ConnectionManager.ts

```diff
@@ -49,27 +48,26 @@ export class ConnectionManager extends (EventEmitter as { new(): StateEmitter })
    *
    * @returns a list of the networks that are currently connected
    */
-  get registeredClients(): string[] {
-    return this.#networks.map(s => s.name);
+  get registeredClients(): Client[] {
+    return this.#networks.map((s: Network) => ({name: s.name, status: s.status}));
```

Comment:
The comment above says that it's returning a list of networks, should we update the comment?

Another question: Is it possible that the same client is used in more than one network? If that was the case: shouldn't this function return a unique list of clients?

Reply:
As discussed - I understand the confusion. I already altered the words to make more sense